tests/util/gentle-register-property.js

```diff
@@ -6,15 +6,16 @@
  * @param {string} [meta.syntax] - Property syntax.
  * @param {boolean} [meta.inherits] - Whether the property inherits.
  * @param {*} [meta.initialValue] - Initial value.
+ * @param {Window} [window] - The window to check in.
  * @returns {boolean} - Whether the property was successfully registered.
  */
-export default function gentleRegisterProperty (property, meta = {}, global = globalThis) {
-	if (!property.startsWith("--") || !global.CSS?.registerProperty) {
+export default function gentleRegisterProperty (property, meta = {}, window = globalThis) {
+	if (!property.startsWith("--") || !window.CSS?.registerProperty) {
```

Comment:
Why the change?

Reply:
Just to be consistent with [`src`'s `gentleRegisterProperty`](https://github.com/LeaVerou/style-observer/pull/109/files#diff-d5848aa53f74b935d8cb04711a71b56953f88262a5d26db65d7388f170ebf694L29), as well as [`isRegisteredProperty`](https://github.com/LeaVerou/style-observer/blob/main/src/util/is-registered-property.js#L8). I can revert if you prefer.